packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -81,11 +82,11 @@ export function buildDocRefBundleWithAttachment(
   patientId: string,
   attachmentUrl: string,
   mimeType: ConsolidationConversionType
-): Bundle<Resource> {
+): SearchSetBundle<Resource> {
   return {
     resourceType: "Bundle",
     total: 1,
-    type: "collection",
+    type: "searchset",
```

Comment:
We were not returning a consistent type of Bundle:
- JSON return `searchset`
- PDF/HTML returned `collection`